src/current/v24.1/create-procedure.md

```diff
@@ -68,6 +68,67 @@ NOTICE: (1,foo)
 CALL
 ~~~
 
+### Create a stored procedure that uses `OUT` and `INOUT` parameters
+
+The following example uses a combination of `OUT` and `INOUT` parameters to modify a provided value and output the result. An `OUT` parameter returns a value, while an `INOUT` parameter passes an input value and returns a value.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE OR REPLACE PROCEDURE double_triple(INOUT double INT, OUT triple INT) AS
+  $$
+  BEGIN
+    double := double * 2;
+    triple := double * 3;
+  END;
+  $$ LANGUAGE PLpgSQL;
+~~~
+
+When calling a procedure, you need to supply placeholder values for any `OUT` parameters. When [calling a procedure from another routine](#create-a-stored-procedure-that-calls-a-procedure), you should declare variables that will store the results of the `OUT` parameters.
```

Comment:
nit: I'd explicitly mention that it's common to use `NULL` values for `OUT` parameters.